examples/pom.xml

```diff
@@ -276,4 +276,60 @@
 			</build>
 		</profile>
     </profiles>
+    
+    <repositories>
```

Comment:
The examples land into ${eap.home}/quickstarts/camel

I put the extra repos there so that released parent poms can download and there is less differences between community and product sources.  It should not hurt the community builds at all to have extra repos.